controllers/get-articles.controller.js

```diff
@@ -15,4 +15,13 @@ function getArticlesId(request, response, next) {
     
 }
 
-module.exports = getArticlesId
\ No newline at end of file
+function getArticles(request, response, next) {
+    getArticlesModel()
+    .then(result => {
+        return response.status(200).send({ articles: result.rows } );
```

Comment:
You shouldn't need the return key word here.

Also, sending the response on a key of 'articles' to represent that it's articles (plural) which are being returned to the client is good, and a great use of names - well done